src/main/kotlin/au/org/ala/cas/delegated/AttributeParser.kt

```diff
@@ -145,6 +145,8 @@ class GithubAttributeParser(val userAttributes: Map<String, Any>) : AttributePar
             val url = URL(urlStr)
             conn = url.openConnection() as HttpURLConnection
             conn.requestMethod = "GET"
+            var authHeader = "token " + githubAccessToken
```

Comment:
Prefer val for constant variables and string interpolation for readability in Kotlin, eg:
`val authHeader = "token $githubAccessToken"`